src/message-fragmentation.md

```diff
@@ -6,16 +6,22 @@ destination node is responsible for collecting the fragments together and reasse
 
 ## Fragment structure
 
+// TODO it is not exactly clear where this fragment is when looking at `sphinx`. IIUC sphinx payload contains
+// the fragment. (means that some of the point I raised in `sphinx.md` can just refer to this md.
 The 2,048 bytes in a fragment are split as follows:
 
 | Bytes    | Field                   | Description                                                    |
 |----------|-------------------------|----------------------------------------------------------------|
 | 0..15    | `message_id`            | The same for all fragments of a message                        |
 | 16..17   | `num_fragments_minus_1` | Little-endian number of fragments minus one                    |
+// TODO theorical max size of a full message (2^16 + 1) * fragement payload size.
+// TODO with two byte of size, is it really worth it to minus 1, I mean it prevents size 0, but size 0 is not
+// that problematic, it would just mean empty message.
```

Comment:
IIRC I just did this because it made things a bit easier.

Reply:
I don't really think this can be an issue, just it feels simpler to me to avoid this offset, but I did not really check the consequences.